specification/azsadmin/resource-manager/subscriptions/Microsoft.Subscriptions.Admin/preview/2015-11-01/examples/Subscriptions/Get.json

```diff
@@ -1,6 +1,7 @@
 {
   "parameters": {
     "subscriptionId": "20ff7fc3-e762-44dd-bd96-b71116dcdc23",
+    "targetSubscriptionId": "c79389af-4480-48cc-8fa2-ee8ced8e843d",
```

Comment:
minor issue, should we use some neutral ID like 00000000-0000-... in examples?

Reply:
These are from AzureStack Test environments and we bring up 100s azurestack of environments in a day, pls treat this like a random guid, not an issue. thanks for asking

Reply:
as explained , this is not an issue. its just a random guid